### PR TITLE
HyperShift Doc Scaffolding

### PR DESCRIPTION
**Pull request by day0hero**
Adding initial docs

### CODE REVIEW

**Comment by mbaldessari** [comment]
This is an automated message:

You can preview this docs PR at http://365.docs-pr.validatedpatterns.io
Note that they get generated every five minutes, so please wait a bit.

File: modules/comm-attributes.adoc [hunk 1]
```diff
@@ -13,6 +13,8 @@
 //Patterns
 :aeg-pattern: Ansible Edge GitOps pattern
 :aeg: Ansible Edge GitOps
+:hcp: Hosted Control Plane
```
Comment (abhatt-rh) [inline modules/comm-attributes.adoc:16]:
```suggestion
:hcp: hosted control plane
```

**Review by abhatt-rh — COMMENTED** [review]
Hi @day0hero, I added a couple of comments for your consideration. These are mostly around adherence to RH/IBM style guides and using product/component names.

File: content/patterns/hypershift/_index.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+//Module to be included
+//:_content-type: CONCEPT
+//:imagesdir: ../../images
+[id="about-hyp-pattern"]
+= About the {hyp-pattern}
+
+Background::
+
+This pattern is designed and intended to assist with simplifying the deployment of an {hyp} Hosting cluster. When this pattern is deployed, users will be be able to create hostedclusters of their own.
```
[the remark this answers is not in the document]
Comment (day0hero) [reply content/patterns/hypershift/_index.adoc]: /lgtm

**Comment by abhatt-rh** [comment]
Will look into the htmltest failures in a separate PR; don't want to hold merge for those

**Pull request merged**